src/content/docs/tutorial-create-alerts/manage-alert-quality.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: "Manage your alert quality"
+metaDescription: "How to manage the quality of your alerts with New Relic"
+redirects: 
+  - /docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide/
+  - /docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alert-quality-management-tutorial/
+  - /docs/new-relic-solutions/observability-maturity/aqm-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide
+---
+
+Teams suffer from alert fatigue when experiencing high alert volumes and alerts that aren't aligned to business impact. When this happens,incident responders often assume the alerts are false and have no business impact. In turn, they may start to prioritize easy-to-resolve alerts over others or close unresolved incidents to stay within their response time targets. This almost always results in slower incident response and increased scope and severity when major issues occur.
+
+Alert quality management (AQM) focuses on reducing the number of nuisance incidents so that you focus only on alerts with true business impact. This reduces alert fatigue and ensures that you and your team focus your attention on the right places at the right times.
```

Comment:
I don't think AQM should be the subject of the sentence here. Address the reader:

> To break the cycle, you need to improve the quality of your alerts. A policy of alerts quality management (AQM)...

or something along those lines. Put the reader and the problems at the center.